pkg/diff/diff.go

```diff
@@ -3,14 +3,32 @@ package diff
 import (
 	"bufio"
 	"bytes"
+	"errors"
```

Comment:
```suggestion
	"github.com/pkg/errors"
```
Looking the test failures, I realised that we need to import a different package. `errors` doesn't have `Errorf` function.

Reply:
Rather than this - I believe a more apt solution will be: `fmt.Errorf`. 😄